packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -202,10 +202,17 @@ export function useInnerBlocksProps( props = {}, options = {} ) {
 				getBlockSettings,
 				getSectionRootClientId,
 			} = unlock( select( blockEditorStore ) );
-			let _isDropZoneDisabled;
 
 			if ( ! clientId ) {
-				return { isDropZoneDisabled: _isDropZoneDisabled };
+				const sectionRootClientId = getSectionRootClientId();
+				// Disable the root drop zone when zoomed out and the section root client id
+				// is not the root block list (represented by an empty string).
+				// This avoids drag handling bugs caused by having two block lists acting as
+				// drop zones - the actual 'root' block list and the section root.
+				return {
+					isDropZoneDisabled:
+						isZoomOut() && sectionRootClientId !== '',
```

Comment:
@talldan After this merged I kept getting an error on the `wp/6.7` branch that `isZoomOut` wasn't defined. I'm super confused because during testing it all seemed to work fine but I guess `wp/6.7` removed the `isZoomOut` line on 198.

Fix in https://github.com/WordPress/gutenberg/pull/66420